src/main/java/seedu/duke/util/Parser.java

```diff
@@ -432,7 +420,7 @@ public String inputContent() throws StringIndexOutOfBoundsException {
      * @param characters String of character to be removed.
      * @param noOfChar Number of character. 0 to remove new line, 1 to resume typing on the same line.
      */
-    public void deleteLine(StringBuilder commandInput, String characters, int noOfChar) {
+    public static void deleteLine(StringBuilder commandInput, String characters, int noOfChar) {
```

Comment:
noOfChar feels a little hard on the eye. Can I suggest something else?
```suggestion
    public static void deleteLine(StringBuilder commandInput, String characters, int charCount) {
```